doc/topics/releases/3006.rst

```diff
@@ -0,0 +1,137 @@
+| Ubuntu   | https://repo.saltproject.io/salt/py3/ubuntu/  | https://repo.saltproject.io/py3/ubuntu/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Debian   | https://repo.saltproject.io/salt/py3/debian/  | https://repo.saltproject.io/py3/debian/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Raspbian | https://repo.saltproject.io/salt/py3/debian/  | https://repo.saltproject.io/py3/debian/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Fedora   | Hosted on Fedora Repos                        | Hosted on Fedora Repos                  |
++----------+-----------------------------------------------+-----------------------------------------+
+| MacOS    | https://repo.saltproject.io/salt/py3/osx/     | https://repo.saltproject.io/osx/        |
++----------+-----------------------------------------------+-----------------------------------------+
+| Windows  | https://repo.saltproject.io/salt/py3/windows/ |  https://repo.saltproject.io/windows/   |
++----------+-----------------------------------------------+-----------------------------------------+
+
+Note that the Tiamat paths above will not be available until the day of the Phosphorus release.
+
+
+My OS only provides Tiamat packages. What do I do?
+--------------------------------------------------
+If you are using a platform that no longer provides the non-Tiamat packages as shown
+in the previous tables, you will need to update your repo files to point to the new Tiamat packages.
+The new repo paths are provided in the previous tables, but the full instructions will be included
+at https://repo.saltproject.io.
+
+
+My OS uses an unsupported version of Python. What do I do?
+----------------------------------------------------------
+If your OS is using an unsupported version of Python, you will need to use the Tiamat packages.
+
+
+How do I migrate to the Tiamat packages?
+----------------------------------------
+The migration path from the non-Tiamat packages to the Tiamat packages will include:
+
+* Repo File: You need to update your repo file to point to the new repo paths for your platform. After the repo
+  file is updated, upgrade your Salt packages.
+* Pip packages: You need to ensure any 3rd party pip packages are installed in the correct Tiamat path.
+  This can be accomplished in two ways:
+
+  * ``salt pip install <package name>``
+  * Using the ``pip.installed`` Salt state.
+
+To install python packages into the system python environment, user's must now provide the ``pip_bin`` or ``bin_env`` to the pip state module.
+
+For example:
+
+.. code-block:: yaml
+
+    lib-foo:
+      pip.installed:
+        - pip_bin: /usr/bin/pip3
+    lib-bar:
+      pip.installed:
+        - bin_env: /usr/bin/python3
```

Comment:
**Suggestion:** Same suggestion as above where I don't think we should duplicate install content and should instead link out to the install guide.